### PR TITLE
[PSDK] winbase.h: Update Create*() dwCreationFlags value defines

### PR DESCRIPTION
**Pull request by SergeGautherie**
* Add more values.
* Add a Vista check.

### CODE REVIEW

File: sdk/include/psdk/winbase.h [hunk 1]
```diff
@@ -189,13 +191,28 @@ extern "C" {
 #define CREATE_FORCEDOS			0x00002000
 #define BELOW_NORMAL_PRIORITY_CLASS	0x00004000
 #define ABOVE_NORMAL_PRIORITY_CLASS	0x00008000
+#if (_WIN32_WINNT >= _WIN32_WINNT_VISTA)
+#define INHERIT_PARENT_AFFINITY             0x00010000
+// TODO: When waa it deprecated?
```
Comment (HBelusca) [inline sdk/include/psdk/winbase.h:196]: ??

Comment (SergeGautherie) [reply sdk/include/psdk/winbase.h:196]: As in, It's marked as deprecated in SDK 10, but I don't know when that happened (earlier)...

**Comment by SergeGautherie** [comment]
@learn-more, I'm looking for a reviewer...

**Comment by SergeGautherie** [comment]
Added `0x00400000` and `0x00800000`, as comments.

File: sdk/include/psdk/winbase.h [hunk 2]
```diff
@@ -189,13 +191,29 @@ extern "C" {
...
+#if (_WIN32_WINNT >= _WIN32_WINNT_VISTA)
+#define INHERIT_PARENT_AFFINITY             0x00010000
+#define INHERIT_CALLER_PRIORITY             0x00020000 // Deprecated. (As of WDK 7.1 already...)
+#define CREATE_PROTECTED_PROCESS            0x00040000
+#define EXTENDED_STARTUPINFO_PRESENT        0x00080000
+#define PROCESS_MODE_BACKGROUND_BEGIN       0x00100000 // See SetPriorityClass() dwPriorityClass.
+#define PROCESS_MODE_BACKGROUND_END         0x00200000 // See SetPriorityClass() dwPriorityClass.
+#endif // _WIN32_WINNT_VISTA
+/* #define CREATE_SECURE_PROCESS               0x00400000  // Documented on MS Docs, absent in MS file. */
+/* #define Unknown/Inexistent                  0x00800000 */
```
Comment (HBelusca) [inline sdk/include/psdk/winbase.h:203]:
```suggestion
```

File: sdk/include/psdk/winbase.h [hunk 3]
```diff
@@ -189,13 +191,29 @@ extern "C" {
...
 #define ABOVE_NORMAL_PRIORITY_CLASS	0x00008000
+#if (_WIN32_WINNT >= _WIN32_WINNT_VISTA)
+#define INHERIT_PARENT_AFFINITY             0x00010000
+#define INHERIT_CALLER_PRIORITY             0x00020000 // Deprecated. (As of WDK 7.1 already...)
+#define CREATE_PROTECTED_PROCESS            0x00040000
+#define EXTENDED_STARTUPINFO_PRESENT        0x00080000
+#define PROCESS_MODE_BACKGROUND_BEGIN       0x00100000 // See SetPriorityClass() dwPriorityClass.
+#define PROCESS_MODE_BACKGROUND_END         0x00200000 // See SetPriorityClass() dwPriorityClass.
+#endif // _WIN32_WINNT_VISTA
+/* #define CREATE_SECURE_PROCESS               0x00400000  // Documented on MS Docs, absent in MS file. */
```
Comment (HBelusca) [inline sdk/include/psdk/winbase.h:202]: Certainly appeared in some Win10 version?

**Comment by SergeGautherie** [comment]
Eventually updated and committed by Hermès as d7fa429, a27227b and 3736938.

**Comment by HBelusca** [comment]
Oops I didn't remember your pr when I made my commits, sorry my bad 😥

**Pull request closed without merging**